docs/Spark/gems/source-target/advanced/synthetic-data-generator/providers.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Providers
+id: providers
+description: What kind of random data can I generate?
+sidebar_position: 1
+tags:
+  - provider
+  - synthetic
+  - random
+  - fake
+  - mock
+  - data
+  - generator
+---
+
+To generate a new column of random data, select one of the [providers](#providers) below. There are some properties [common](#common-properties) to all providers. If you prefer, provide the same information as a JSON schema.
+
+## Providers
+
+| Data Provider             | Description                                                                                                                                                                                                                                                 |
+| ------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Random Name               | Generates random names. Select Full Name, First Name, Last Name as the sub-types.                                                                                                                                                                           |
+| Random Address            | Generates random addresses                                                                                                                                                                                                                                  |
```

Comment:
Some of the ends of these lines have periods. Others do not. I suggest adding periods for all.